source/features/linkify-notification-page-numbers.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'dom-chef';
+import select from 'select-dom';
+import * as pageDetect from 'github-url-detection';
+
+import {wrap} from '../helpers/dom-utils';
+import features from '../feature-manager';
+import looseParseInt from '../helpers/loose-parse-int';
+
+function init(): void {
+	const nextButtonURL = new URL(select('a[data-hotkey="ArrowRight"]')!.href);
+	const nextButtonSearch = new URLSearchParams(nextButtonURL.search);
+	const lastNotificationPage = select('.js-notifications-list-paginator-counts')!.lastChild!;
+	const lastNotificationPageNumber = looseParseInt(lastNotificationPage);
+	const lastCursor = Math.floor(lastNotificationPageNumber / 50) * 50;
+	nextButtonSearch.set('after', btoa('cursor:' + String(lastCursor)));
+	nextButtonURL.search = nextButtonSearch.toString();
+	wrap(lastNotificationPage, <a href={nextButtonURL.href}/>);
```

Comment:
Wrapping the whole node with "of" and whitespace looks ugly, only the number should be linkified. Also the underling should always be visible

Reply:
>Also the underling should always be visible
?

Reply:
Is the underline always visible or only on hover. 

It should always be visible. I don't know how else to phrase it.

Reply:
Done I meant to write `underling ?` but look like I forgot that word.

Changed